### PR TITLE
Fix entry point

### PR DESCRIPTION
**Pull request by tbachman**
Commit f811b830bc38be43e566ab34d5c38a2266c1fe0f changed
the module name, but not the entry point. This fixes the
entry point.

### CODE REVIEW

**Comment by tbachman** [comment]
Good suggestion -- pushed new update, with this change.

File: opflexagent/utils/bridge_managers/vpp_manager.py [hunk 1]
```diff
@@ -33,6 +34,7 @@ def __init__(self):
     def initialize(self, host, conf, agent_state):
         self.int_br_device_count = 0
         vpp_config = conf.VPP
+        agent_state['agent_type'] = ofcst.AGENT_TYPE_OPFLEX_OVS
```
Comment (amittbose) [inline opflexagent/utils/bridge_managers/vpp_manager.py]: this should be AGENT_TYPE_OPFLEX_VPP ?

Comment (tbachman) [reply opflexagent/utils/bridge_managers/vpp_manager.py]: Definitely :-(. Will respin, and thanks for catching!

**Comment by tbachman** [comment]
Pushed new version, addressing reviewer comments (and my own shortcomings ;-) ).

**Pull request merged**